<ui version="4.0" >
 <author>Alexander Saal &lt;alex.saal@gmx.de></author>
 <class>UiDocScannerBase</class>
 <widget class="QWidget" name="UiDocScannerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scan documents</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxCustomer" >
     <property name="title" >
      <string>Documents</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelCustomer_2" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Addressbook:</string>
          </property>
          <property name="buddy" >
           <cstring>cbxCustomer</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxAddressbook" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="labelCustomer" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Customer:</string>
          </property>
          <property name="buddy" >
           <cstring>cbxCustomer</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="cbxCustomer" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QGroupBox" name="groupBoxImageOption" >
        <property name="title" >
         <string>Imageoption</string>
        </property>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QRadioButton" name="rBtnSaveImageBlob" >
           <property name="text" >
            <string>Save image into database (BLOB)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QRadioButton" name="rBtnSaveImageFile" >
           <property name="text" >
            <string>Save image as file</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>623</width>
          <height>111</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnScan" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>&amp;Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbort" >
       <property name="text" >
        <string>&amp;Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxCustomer</tabstop>
  <tabstop>btnScan</tabstop>
  <tabstop>btnAbort</tabstop>
 </tabstops>
 <includes/>
 <resources>
  <include location="../res/docscannerplugin.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnAbort</sender>
   <signal>clicked()</signal>
   <receiver>UiDocScannerBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>652</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
